CanHazFunny/CanHazFunny.Tests/OutputServiceTests.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using Xunit;
+using System.IO;
+
+namespace CanHazFunny.Tests;
+
+public class OutputServiceTests
+{
+    [Fact]
+    public void DisplayJoke_ThrowsArgumentNull_Success()
+    {
+        OutputService displayJoke = new();
+        Assert.Throws<ArgumentNullException>(() => displayJoke.DisplayJoke(null!));
+    }
+
+    [Fact]
+    public void DisplayJoke_JokeDisplays_Success()
+    {
+        MockOutput mockClass = new();
+        mockClass.DisplayJoke("knock knock joke");
+
+        Assert.Equal("knock knock joke", mockClass.GetJoke());
+    }
+
+    [Fact]
+    public void DisplayJoke_ValidJoke_WritesToConsole()
+    {
+        // Arrange
+        string joke = "Funny joke";
+        OutputService jokeDisplay = new();
+        StringWriter stringWriter = new();
+        Console.SetOut(stringWriter);
+
+        // Act
+        jokeDisplay.DisplayJoke(joke);
+
+        // Assert
+        Assert.Contains(joke, stringWriter.ToString());
```

Comment:
```suggestion
        Assert.Equal<string>(joke + Enviroment.NewLine, stringWriter.ToString());
```
In my opinion, Equal with newline is better than contains and it makes sure that it is the same string rather than making sure it only contains it. Although I understand that due to how the method is written, it doesn't technically cause issues. Mark said this was okay and to disable the warning if you decided to change it.

Reply:
Oh awesome! thanks! But here we decided to go for simplicity while testing here and just check whether the command line simply contains the joke. We did not really want to test the entire command line output.